projects/s-js-utils/src/lib/to-csv.ts

```diff
@@ -0,0 +1,21 @@
+import { toString } from "micro-dash";
+
+export function toCsv(data: any[][]) {
+  return data
+    .map((row) => row.map((cell) => escape(cell)).join(","))
+    .join("\n");
+}
+
+function escape(value: any) {
+  const string = toString(value);
+
+  if (/["|,|\n|\r]/.test(string)) {
+    return wrapWithQuotes(string.replace(/"/g, `""`));
+  } else {
+    return string;
+  }
+}
+
+function wrapWithQuotes(string: string) {
+  return `"${string}"`;
```

Comment:
`[green]`
This implementation is shorter than the name of the function. Maybe inline it?